target/snitch_cluster/README.md

```diff
@@ -31,6 +31,9 @@ vcs-2020.12 make bin/snitch_cluster.vcs
 
 These commands compile the RTL sources respectively in `work-vlt`, `work-vsim` and `work-vcs`. Additionally, common C++ testbench sources (e.g. the [frontend server (fesvr)](https://github.com/riscv-software-src/riscv-isa-sim)) are compiled under `work`. Each command will also generate a script or an executable (e.g. `bin/snitch_cluster.vsim`) which you can invoke to simulate the hardware. We will see how to do this in a later section.
 
+### Building the Banshee simulator
+Instead of running an RTL simulation, you can use our instruction-accuracte simulator called `banshee`. To install the simulator, please follow the instructions of the Banshee repository: [https://github.com/pulp-platform/banshee](https://github.com/pulp-platform/banshee).
```

Comment:
Typo: accurate

Reply:
Would it be possible to have Banshee installed on IIS machines? If not, I would add "For Docker container users, Banshee is already installed.", otherwise we can remove this altogether

Reply:
Okay, so just the typo to fix then we can resolve this.